client/my-sites/email/form/mailboxes/types.ts

```diff
@@ -119,6 +119,7 @@ abstract class MailboxFormFields implements IBaseMailboxFormFields {
 class GoogleMailboxFormFields extends MailboxFormFields implements IGoogleMailboxFormFields {
 	firstName? = new TextMailboxFormField( FIELD_FIRSTNAME );
 	lastName? = new TextMailboxFormField( FIELD_LASTNAME );
+	alternativeEmail? = new TextMailboxFormField( FIELD_ALTERNATIVE_EMAIL );
```

Comment:
Could we call this recovery email instead? Or maybe that's the name we use for Titan (because that's how they call it in their API endpoints)? I'm just worried that using both names interchangeably may confuse us (it certainly did in the past!).

Reply:
Titan calls it as Alternate Email too :) Also I've called it as alternative email to share the same behaviour (hide it when its prepopulated)